app/components/Modal/ConfirmModal.js

```diff
@@ -31,20 +34,26 @@ export const ConfirmModal = ({
         <Button disabled={disabled} onClick={onConfirm}>
           Ja
         </Button>
+        <p style={{ color: 'red' }}>{errorMessage} </p>
       </div>
     </div>
   </div>
 );
 
 type State = {
   modalVisible: boolean,
-  working: boolean
+  working: boolean,
```

Comment:
Probably something for a separate PR, but could we make stuff like `working` use cancellable promises instead, and cancel promises when the component unmounts instead of having to keep track of a separate bool? 

Something along the lines of https://reactjs.org/blog/2015/12/16/ismounted-antipattern.html

Reply:
Hmm...

I guess that would be a bit overkill for this... We know when the modal will unmount (due to redirect etc), and when the modal have to do it by it self, and we can set `closeOnConfirm` and `closeOnCancel` according to that.

The best would be to use modals like we currently use Toasts, but that is way out of scope now..